src/components/Components.js

```diff
@@ -207,9 +207,9 @@ const WorkletIndicator = () => {
 
 // Check if AudioWorklet is available.
 function _detectAudioWorklet() {
-  let context = new OfflineAudioContext(1, 1, 44100);
+  let context = window.OfflineAudioContext && new OfflineAudioContext(1, 1, 44100);
```

Comment:
1. I have no idea about the current status of Safari. Did they remove "webkit" prefix?
2. Why do we have to check if OfflineAudioContext is valid? How does this help us detect Safari?

Reply:
This code doesn't necessarily fix safari, but it does prevent a TypeError that occurs when referencing a property/variable.  This would also fix detection of AudioWorklet on older browsers where the `OfflineAudioContect` property does not exist on window.

For example, `const a = FooBar` will throw a TypeError on all browsers, but `const a = window.FooBar` will not

Reply:
I am confused. OfflineAudioContext exists on every browser. I think the clue that you want to check is the webkit prefix.

Thanks for the suggestion, but I prefer something more explicit and less magical. If something bad happens, it's better to throw, catch and display a helpful error message.

Reply:
I hope we can expand l.210 with some comments. (like comments from @AnthumChris will be helpful when we read this code in the future)

cc/ @max-vogler